rest-service/manager_rest/test/endpoints/test_utils.py

```diff
@@ -95,6 +97,104 @@ def mock_get_platform():
                 self.assertTrue(
                     plugin_installable_on_current_platform(plugin))
 
+    def test_tracing_wrapper_does_nothing(self):
+        # Dummy call to invoke the tracer initialization if enabled.
+        self.app.get('/')
+
+        foo = MagicMock(return_value='something')
+        foo.__name__ = 'foo'
+        app_patcher = patch('flask.current_app')
+        curr_span_patcher = patch(
+            'opentracing_instrumentation.request_context.get_current_span')
+        span_ctx_patcher = patch(
+            'opentracing_instrumentation.request_context.span_in_context')
+        current_app = app_patcher.start()
+        curr_span = curr_span_patcher.start()
+        span_ctx_patcher.start()
+
+        f = traces()(foo)
+        current_app.tracer = False
+        r = f('bar', k='v')
+        self.assertEqual(r, 'something')
+        foo.assert_called_once_with('bar', k='v')
+        curr_span.assert_not_called()
+
+        app_patcher.stop()
+        curr_span_patcher.stop()
+        span_ctx_patcher.stop()
+
+
+class TestUtilsWithTracingTestCase(base_test.BaseServerTestCase):
+    """Test the tracing wrapper when tracing is enabled.
+    """
+    _tracing_endpoint_ip = 'some_ip'
```

Comment:
maybe create a test class of yourself

Reply:
I kinda disagree with creating a new test class since the changes made to `BaseServerTestCase` were not that big. Also, I need a different setup therefore I don't think it'll save any work here.

Reply:
There too much copied code

Reply:
I'll need help with this one then